invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/index.js

```diff
@@ -6,14 +6,15 @@
  * Invenio is free software; you can redistribute it and/or modify it
  * under the terms of the MIT License; see LICENSE file for more details.
  */
+import _ from "lodash";
```

Comment:
Can you also import the individual functions you need from lodash instead of the whole library, or even, are they needed at all?

Reply:
thanks for spotting this, yes the individual functions can be importer, and I will do that here